### ISSUE TITLE
FusedBatchNorm problem during benchmark test

### ISSUE THREAD

**Issue by DavideCatto** [tensorflow/tensorflow#21475]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Windows 10
- **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**: None
- **TensorFlow installed from (source or binary)**: 1.9
- **TensorFlow version (use command below)**: 1.9
- **Python version**: 3.6
- **Bazel version (if compiling from source)**: 0.11.1 (for covert and benchmark the model)
- **GCC/Compiler version (if compiling from source)**: None, installed from build
- **CUDA/cuDNN version**: Not used
- **GPU model and memory**: Not used
- **Exact command to reproduce**:

### Problem description
I'm trying to convert a piece of mobilenetV1 (using slim model) from the master repository [https://github.com/tensorflow/models/blob/master/research/slim/nets/mobilenet_v1.md](url); In particular I have added 2 simple convolution after the _MobilenetV1/Conv2d_8_pointwise/Conv2D_. Obviously the number of parameters is more less than the original MobilenetV1. Without train the model, I have initialized all the variables and save them into a freezed model. Now If I visualize the model with tensorboard is the same of that downloaded from the link above until the _MobilenetV1/Conv2d_8_pointwise/Conv2D_ (It compare the Conv2D and the FusedBatchNorm, Gamma, Beta, var and so on). The problems cames when I try to benchmark the model using bazel: the original one is more faster than mine _(FLOPs estimated 1.14B VS 3.31B)_ and also in summaries of the original model the _FusedBatchNorm_ Node type doesn't compare instead in mine model yes. Where am I doing wrong? And there is also the possibility of combining the conv2D and the batch norm layer?

### logs

------------------------- ORIGINAL MOBILENETV1 ---------------------------------------------------------------

>...stat_summarizer.cc:468] 	             [Node type]	  [count]	  [avg ms]	    [avg %]	    [cdf %]	  [mem KB]	[times called]
...stat_summarizer.cc:468] 	                  Conv2D	       15	    52.027	    56.168%	    56.168%	 12447.652	       15
...stat_summarizer.cc:468] 	   DepthwiseConv2dNative	       13	    15.413	    16.640%	    72.808%	  7905.664	       13
...stat_summarizer.cc:468] 	                     **Mul**	       27	    12.018	    12.975%	    85.783%	     0.000	       27
...stat_summarizer.cc:468] 	                     **Add**	       27	    11.806	    12.746%	    98.529%	     0.000	       27
...stat_summarizer.cc:468] 	                   **Relu6**	       27	     1.163	     1.256%	    99.784%	     0.000	       27
...stat_summarizer.cc:468] 	                   Const	       83	     0.120	     0.130%	    99.914%	     0.000	       83
...stat_summarizer.cc:468] 	                 Softmax	        1	     0.027	     0.029%	    99.943%	     0.000	        1
...stat_summarizer.cc:468] 	                 AvgPool	        1	     0.022	     0.024%	    99.967%	     4.096	        1
...stat_summarizer.cc:468] 	                    NoOp	        1	     0.007	     0.008%	    99.974%	     0.000	        1
...stat_summarizer.cc:468] 	                 BiasAdd	        1	     0.007	     0.008%	    99.982%	     0.000	        1
...stat_summarizer.cc:468] 	                 Squeeze	        1	     0.006	     0.006%	    99.988%	     0.000	        1
...stat_summarizer.cc:468] 	                 _Retval	        1	     0.005	     0.005%	    99.994%	     0.000	        1
...stat_summarizer.cc:468] 	                    _Arg	        1	     0.004	     0.004%	    99.998%	     0.000	        1
...stat_summarizer.cc:468] 	                Identity	        1	     0.002	     0.002%	   100.000%	     0.000	        1
--------------------------------------------------------------------------------------------------------------

---------------------------- MY MODEL ------------------------------------------------------------------------
>...stat_summarizer.cc:468] 	             [Node type]	  [count]	  [avg ms]	    [avg %]	    [cdf %]	  [mem KB]	[times called]
...stat_summarizer.cc:468] 	          **FusedBatchNorm**	       18	   151.477	    48.975%	    48.975%	    72.704	       18
...stat_summarizer.cc:468] 	                  Conv2D	       11	   138.036	    44.630%	    93.605%	 15457.344	       11
...stat_summarizer.cc:468] 	   DepthwiseConv2dNative	        8	    18.148	     5.868%	    99.472%	  9300.352	        8
...stat_summarizer.cc:468] 	                    Relu	       18	     1.581	     0.511%	    99.984%	     0.000	       18
...stat_summarizer.cc:468] 	                   Const	       29	     0.038	     0.012%	    99.996%	     0.000	       29
...stat_summarizer.cc:468] 	                 _Retval	        1	     0.005	     0.002%	    99.997%	     0.000	        1
...stat_summarizer.cc:468] 	                    _Arg	        1	     0.004	     0.001%	    99.999%	     0.000	        1
...stat_summarizer.cc:468] 	                    NoOp	        1	     0.004	     0.001%	   100.000%	     0.000	        1
--------------------------------------------------------------------------------------------------------------
OK, I have noted the differen Relu activation but I do not think that's the problem for the fusedbatchnorm layer..

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tatatodd: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tatatodd: It has been 30 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tatatodd: It has been 44 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tatatodd: It has been 59 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tatatodd: It has been 75 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tatatodd: It has been 90 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tatatodd: It has been 105 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tatatodd: It has been 120 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
